Jenkinsfile

```diff
@@ -1,22 +1,5 @@
-
 def rocmnode(name) {
-    def node_name = 'rocmtest'
-    if(name == 'vega') {
-        node_name = 'rocmtest && vega';
-    } else if(name == 'vega10') {
-        node_name = 'rocmtest && vega10';
-    } else if(name == 'vega20') {
-        node_name = 'rocmtest && vega20';
-    } else if(name == 'gfx908') {
-        node_name = 'gfx908';
-    } else {
-        node_name = name
-    }
-    return node_name
-}
-
-def default_image_name() {
-    return 'miopen-v2'
+    return 'rocmtest && miopen && ' + name
```

Comment:
This should be `(rocmtest || miopen)` so you can allocate nodes just for miopen in the future.

Reply:
The idea is to allocate nodes exclusively for miopen by removing other labels (like `migraphx`) from the nodes. From the PR description:

> [Side note] All relevant CI nodes updated with 'nogpu', 'miopen and 'rocmtest' labels:
> - 'nogpu' is set on nodes that are allowed to run tests that do not require a GPU. Currently, these are all nodes except MI100. 
> - 'rocmtest' means that the node belongs to the "ROCmSoftwarePlatform" github organization (which now includes MIOpen, AMDMIGraphX, MIOpenGEMM, MIOpenTensile).
> - 'miopen' means that the node is allowed to run MIOpen tests.

Reply:
MIGraphX uses `rocmtest || migraphx` so you would need to remove both migraphx label and the rocmtest label.

Reply:
Thanks! Currently I do not plan to be greedy and allocate nodes for MIOpen exclusively.

However -- I would appreciate if you find the hierarchic approach in this file more logical and borrow it for MIGraphX.